roles/zabbix_agent/tasks/Debian.yml

```diff
@@ -61,7 +61,7 @@
       Types: deb{{ ' deb-src' if zabbix_repo_deb_include_deb_src }}
       Enabled: yes
       URIs: {{ zabbix_repo_deb_url }}
-      Suites: {{ ansible_distribution_release }}
+      Suites: {{ zabbix_agent_distribution_release }}
```

Comment:
There's no default provided for this, it's gonna break for everyone that hasn't defined this variable. It was cleaned out by c2a32d5ea130eadbf4c4a0b6160757cd9a7e8ac9